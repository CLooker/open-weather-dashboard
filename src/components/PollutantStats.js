import React from 'react';

const PollutantStats = ({ type, chem }) => {
  switch (type) {
    case 'co':
    case 'so2':
      return (
        <div key={type} className="pollutant-stats">
          <div>
            <strong>{type.toUpperCase()}</strong>
          </div>
          <div>Value: {chem.value}</div>
          <div>Pressure: {chem.pressure} hPa </div>
          <div>Precision: {chem.precision}</div>
        </div>
      );
    case 'o3':
      return (
        <div key={type}>
          <div>{type.toUpperCase()}</div>
          <div>Value: {chem.value}</div>
        </div>
      );
    case 'no2':
      return (
        <div key={type}>
          <div>{type.toUpperCase()}</div>
          <div>Value: {chem.value}</div>
          <div>Precision: {chem.precision}</div>
        </div>
      );
    default:
      return null;
  }
};

export default PollutantStats;
