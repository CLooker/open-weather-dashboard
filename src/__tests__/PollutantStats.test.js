import React from 'react';
import { shallow, mount } from 'enzyme';
import PollutantStats from '../components/PollutantStats';

describe('PollutantStats', () => {
  const props = [
    {
      type: 'co',
      chem: {
        value: 100,
        pressure: 1000,
        precision: 32924
      }
    },
    {
      type: 'so2',
      chem: {
        value: 132,
        pressure: 2000,
        precision: 23234
      }
    },
    {
      type: 'o3',
      chem: {
        value: 2349234
      }
    },
    {
      type: 'no2',
      chem: {
        value: 34539405,
        precision: 2345234
      }
    }
  ];

  describe('initialization', () => {
    it('should render correctly', () => {
      props.forEach(prop => {
        const component = shallow(<PollutantStats {...prop} />);
        expect(component).toMatchSnapshot();
      });
    });
  });
});
