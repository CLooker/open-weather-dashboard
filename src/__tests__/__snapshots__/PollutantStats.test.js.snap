// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PollutantStats initialization should render correctly 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <PollutantStats
    chem={
      Object {
        "precision": 32924,
        "pressure": 1000,
        "value": 100,
      }
    }
    type="co"
  />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": "co",
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <strong>
          CO
        </strong>,
        <div>
          <span>
            <h5>
              Value:
            </h5>
             
            <p>
              100
            </p>
          </span>
          <span>
            <h5>
              Pressure:
            </h5>
             
            <p>
              1000
               hPa
            </p>
          </span>
          <span>
            <h5>
              Precision:
            </h5>
             
            <p>
              32924
            </p>
          </span>
        </div>,
      ],
      "className": "pollutant-stats",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "CO",
        },
        "ref": null,
        "rendered": "CO",
        "type": "strong",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <span>
              <h5>
                Value:
              </h5>
               
              <p>
                100
              </p>
            </span>,
            <span>
              <h5>
                Pressure:
              </h5>
               
              <p>
                1000
                 hPa
              </p>
            </span>,
            <span>
              <h5>
                Precision:
              </h5>
               
              <p>
                32924
              </p>
            </span>,
          ],
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                <h5>
                  Value:
                </h5>,
                " ",
                <p>
                  100
                </p>,
              ],
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "Value:",
                },
                "ref": null,
                "rendered": "Value:",
                "type": "h5",
              },
              " ",
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": 100,
                },
                "ref": null,
                "rendered": 100,
                "type": "p",
              },
            ],
            "type": "span",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                <h5>
                  Pressure:
                </h5>,
                " ",
                <p>
                  1000
                   hPa
                </p>,
              ],
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "Pressure:",
                },
                "ref": null,
                "rendered": "Pressure:",
                "type": "h5",
              },
              " ",
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": Array [
                    1000,
                    " hPa",
                  ],
                },
                "ref": null,
                "rendered": Array [
                  1000,
                  " hPa",
                ],
                "type": "p",
              },
            ],
            "type": "span",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                <h5>
                  Precision:
                </h5>,
                " ",
                <p>
                  32924
                </p>,
              ],
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "Precision:",
                },
                "ref": null,
                "rendered": "Precision:",
                "type": "h5",
              },
              " ",
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": 32924,
                },
                "ref": null,
                "rendered": 32924,
                "type": "p",
              },
            ],
            "type": "span",
          },
        ],
        "type": "div",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": "co",
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <strong>
            CO
          </strong>,
          <div>
            <span>
              <h5>
                Value:
              </h5>
               
              <p>
                100
              </p>
            </span>
            <span>
              <h5>
                Pressure:
              </h5>
               
              <p>
                1000
                 hPa
              </p>
            </span>
            <span>
              <h5>
                Precision:
              </h5>
               
              <p>
                32924
              </p>
            </span>
          </div>,
        ],
        "className": "pollutant-stats",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "CO",
          },
          "ref": null,
          "rendered": "CO",
          "type": "strong",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <span>
                <h5>
                  Value:
                </h5>
                 
                <p>
                  100
                </p>
              </span>,
              <span>
                <h5>
                  Pressure:
                </h5>
                 
                <p>
                  1000
                   hPa
                </p>
              </span>,
              <span>
                <h5>
                  Precision:
                </h5>
                 
                <p>
                  32924
                </p>
              </span>,
            ],
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  <h5>
                    Value:
                  </h5>,
                  " ",
                  <p>
                    100
                  </p>,
                ],
              },
              "ref": null,
              "rendered": Array [
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "Value:",
                  },
                  "ref": null,
                  "rendered": "Value:",
                  "type": "h5",
                },
                " ",
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": 100,
                  },
                  "ref": null,
                  "rendered": 100,
                  "type": "p",
                },
              ],
              "type": "span",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  <h5>
                    Pressure:
                  </h5>,
                  " ",
                  <p>
                    1000
                     hPa
                  </p>,
                ],
              },
              "ref": null,
              "rendered": Array [
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "Pressure:",
                  },
                  "ref": null,
                  "rendered": "Pressure:",
                  "type": "h5",
                },
                " ",
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": Array [
                      1000,
                      " hPa",
                    ],
                  },
                  "ref": null,
                  "rendered": Array [
                    1000,
                    " hPa",
                  ],
                  "type": "p",
                },
              ],
              "type": "span",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  <h5>
                    Precision:
                  </h5>,
                  " ",
                  <p>
                    32924
                  </p>,
                ],
              },
              "ref": null,
              "rendered": Array [
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "Precision:",
                  },
                  "ref": null,
                  "rendered": "Precision:",
                  "type": "h5",
                },
                " ",
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": 32924,
                  },
                  "ref": null,
                  "rendered": 32924,
                  "type": "p",
                },
              ],
              "type": "span",
            },
          ],
          "type": "div",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
    "disableLifecycleMethods": true,
  },
}
`;

exports[`PollutantStats initialization should render correctly 2`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <PollutantStats
    chem={
      Object {
        "precision": 23234,
        "pressure": 2000,
        "value": 132,
      }
    }
    type="so2"
  />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": "so2",
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <strong>
          SO2
        </strong>,
        <div>
          <span>
            <h5>
              Value:
            </h5>
             
            <p>
              132
            </p>
          </span>
          <span>
            <h5>
              Pressure:
            </h5>
             
            <p>
              2000
               hPa
            </p>
          </span>
          <span>
            <h5>
              Precision:
            </h5>
             
            <p>
              23234
            </p>
          </span>
        </div>,
      ],
      "className": "pollutant-stats",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "SO2",
        },
        "ref": null,
        "rendered": "SO2",
        "type": "strong",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <span>
              <h5>
                Value:
              </h5>
               
              <p>
                132
              </p>
            </span>,
            <span>
              <h5>
                Pressure:
              </h5>
               
              <p>
                2000
                 hPa
              </p>
            </span>,
            <span>
              <h5>
                Precision:
              </h5>
               
              <p>
                23234
              </p>
            </span>,
          ],
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                <h5>
                  Value:
                </h5>,
                " ",
                <p>
                  132
                </p>,
              ],
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "Value:",
                },
                "ref": null,
                "rendered": "Value:",
                "type": "h5",
              },
              " ",
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": 132,
                },
                "ref": null,
                "rendered": 132,
                "type": "p",
              },
            ],
            "type": "span",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                <h5>
                  Pressure:
                </h5>,
                " ",
                <p>
                  2000
                   hPa
                </p>,
              ],
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "Pressure:",
                },
                "ref": null,
                "rendered": "Pressure:",
                "type": "h5",
              },
              " ",
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": Array [
                    2000,
                    " hPa",
                  ],
                },
                "ref": null,
                "rendered": Array [
                  2000,
                  " hPa",
                ],
                "type": "p",
              },
            ],
            "type": "span",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                <h5>
                  Precision:
                </h5>,
                " ",
                <p>
                  23234
                </p>,
              ],
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "Precision:",
                },
                "ref": null,
                "rendered": "Precision:",
                "type": "h5",
              },
              " ",
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": 23234,
                },
                "ref": null,
                "rendered": 23234,
                "type": "p",
              },
            ],
            "type": "span",
          },
        ],
        "type": "div",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": "so2",
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <strong>
            SO2
          </strong>,
          <div>
            <span>
              <h5>
                Value:
              </h5>
               
              <p>
                132
              </p>
            </span>
            <span>
              <h5>
                Pressure:
              </h5>
               
              <p>
                2000
                 hPa
              </p>
            </span>
            <span>
              <h5>
                Precision:
              </h5>
               
              <p>
                23234
              </p>
            </span>
          </div>,
        ],
        "className": "pollutant-stats",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "SO2",
          },
          "ref": null,
          "rendered": "SO2",
          "type": "strong",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <span>
                <h5>
                  Value:
                </h5>
                 
                <p>
                  132
                </p>
              </span>,
              <span>
                <h5>
                  Pressure:
                </h5>
                 
                <p>
                  2000
                   hPa
                </p>
              </span>,
              <span>
                <h5>
                  Precision:
                </h5>
                 
                <p>
                  23234
                </p>
              </span>,
            ],
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  <h5>
                    Value:
                  </h5>,
                  " ",
                  <p>
                    132
                  </p>,
                ],
              },
              "ref": null,
              "rendered": Array [
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "Value:",
                  },
                  "ref": null,
                  "rendered": "Value:",
                  "type": "h5",
                },
                " ",
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": 132,
                  },
                  "ref": null,
                  "rendered": 132,
                  "type": "p",
                },
              ],
              "type": "span",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  <h5>
                    Pressure:
                  </h5>,
                  " ",
                  <p>
                    2000
                     hPa
                  </p>,
                ],
              },
              "ref": null,
              "rendered": Array [
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "Pressure:",
                  },
                  "ref": null,
                  "rendered": "Pressure:",
                  "type": "h5",
                },
                " ",
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": Array [
                      2000,
                      " hPa",
                    ],
                  },
                  "ref": null,
                  "rendered": Array [
                    2000,
                    " hPa",
                  ],
                  "type": "p",
                },
              ],
              "type": "span",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  <h5>
                    Precision:
                  </h5>,
                  " ",
                  <p>
                    23234
                  </p>,
                ],
              },
              "ref": null,
              "rendered": Array [
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "Precision:",
                  },
                  "ref": null,
                  "rendered": "Precision:",
                  "type": "h5",
                },
                " ",
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": 23234,
                  },
                  "ref": null,
                  "rendered": 23234,
                  "type": "p",
                },
              ],
              "type": "span",
            },
          ],
          "type": "div",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
    "disableLifecycleMethods": true,
  },
}
`;

exports[`PollutantStats initialization should render correctly 3`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <PollutantStats
    chem={
      Object {
        "value": 2349234,
      }
    }
    type="o3"
  />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": "o3",
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <div>
          O3
        </div>,
        <div>
          Value: 
          2349234
        </div>,
      ],
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "O3",
        },
        "ref": null,
        "rendered": "O3",
        "type": "div",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            "Value: ",
            2349234,
          ],
        },
        "ref": null,
        "rendered": Array [
          "Value: ",
          2349234,
        ],
        "type": "div",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": "o3",
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <div>
            O3
          </div>,
          <div>
            Value: 
            2349234
          </div>,
        ],
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "O3",
          },
          "ref": null,
          "rendered": "O3",
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              "Value: ",
              2349234,
            ],
          },
          "ref": null,
          "rendered": Array [
            "Value: ",
            2349234,
          ],
          "type": "div",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
    "disableLifecycleMethods": true,
  },
}
`;

exports[`PollutantStats initialization should render correctly 4`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <PollutantStats
    chem={
      Object {
        "precision": 2345234,
        "value": 34539405,
      }
    }
    type="no2"
  />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": "no2",
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <div>
          NO2
        </div>,
        <div>
          Value: 
          34539405
        </div>,
        <div>
          Precision: 
          2345234
        </div>,
      ],
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "NO2",
        },
        "ref": null,
        "rendered": "NO2",
        "type": "div",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            "Value: ",
            34539405,
          ],
        },
        "ref": null,
        "rendered": Array [
          "Value: ",
          34539405,
        ],
        "type": "div",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            "Precision: ",
            2345234,
          ],
        },
        "ref": null,
        "rendered": Array [
          "Precision: ",
          2345234,
        ],
        "type": "div",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": "no2",
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <div>
            NO2
          </div>,
          <div>
            Value: 
            34539405
          </div>,
          <div>
            Precision: 
            2345234
          </div>,
        ],
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "NO2",
          },
          "ref": null,
          "rendered": "NO2",
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              "Value: ",
              34539405,
            ],
          },
          "ref": null,
          "rendered": Array [
            "Value: ",
            34539405,
          ],
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              "Precision: ",
              2345234,
            ],
          },
          "ref": null,
          "rendered": Array [
            "Precision: ",
            2345234,
          ],
          "type": "div",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
    "disableLifecycleMethods": true,
  },
}
`;
